extensions/mssql/src/objectManagement/ui/databaseDialog.ts

```diff
@@ -195,6 +231,136 @@ export class DatabaseDialog extends ObjectManagementDialogBase<Database, Databas
 	}
 	//#endregion
 
+	//#region Database Properties - Options Tab
+	private initializeOptionsGeneralSection(): void {
+		let containers: azdata.Component[] = [];
+		let collationDropbox = this.createDropdown(localizedConstants.CollationText, async (newValue) => {
+			this.objectInfo.collationName = newValue as string;
+		}, this.viewInfo.collationNames, this.objectInfo.collationName);
+		containers.push(this.createLabelInputContainer(localizedConstants.CollationText, collationDropbox));
+
+		let displayOptionsArray = this.viewInfo.recoveryModels.length === 0 ? [this.objectInfo.recoveryModel] : this.viewInfo.recoveryModels;
+		let isEnabled = this.viewInfo.recoveryModels.length === 0 ? false : true;
+		let recoveryDropbox = this.createDropdown(localizedConstants.RecoveryModelText, async (newValue) => {
+			this.objectInfo.recoveryModel = newValue as string;
+		}, displayOptionsArray, this.objectInfo.recoveryModel, isEnabled);
+		containers.push(this.createLabelInputContainer(localizedConstants.RecoveryModelText, recoveryDropbox));
+
+		let compatibilityDropbox = this.createDropdown(localizedConstants.CompatibilityLevelText, async (newValue) => {
+			this.objectInfo.compatibilityLevel = newValue as string;
+		}, this.viewInfo.compatibilityLevels, this.objectInfo.compatibilityLevel);
+		containers.push(this.createLabelInputContainer(localizedConstants.CompatibilityLevelText, compatibilityDropbox));
+
+		displayOptionsArray = this.viewInfo.containmentTypes.length === 0 ? [this.objectInfo.containmentType] : this.viewInfo.containmentTypes;
+		isEnabled = this.viewInfo.containmentTypes.length === 0 ? false : true;
+		let containmentDropbox = this.createDropdown(localizedConstants.ContainmentTypeText, async (newValue) => {
+			this.objectInfo.containmentType = newValue as string;
+		}, displayOptionsArray, this.objectInfo.containmentType, isEnabled);
+		containers.push(this.createLabelInputContainer(localizedConstants.ContainmentTypeText, containmentDropbox));
+
+		const optionsGeneralSection = this.createGroup('', containers, true, true);
+		this.optionsTabSectionsContainer.push(optionsGeneralSection);
+	}
+
+	private initializeAutomaticSection(): void {
+		this.autoCreateIncrementalStatisticsInput = this.createDropdown(localizedConstants.AutoCreateIncrementalStatisticsText, async (newValue) => {
+			this.objectInfo.autoCreateIncrementalStatistics = Boolean(newValue);
+		}, this.booleanOptionsArray, toUpperCaseFirstChar(String(this.objectInfo.autoCreateIncrementalStatistics)), true);
```

Comment:
This is just a true/false value - correct? This value should be passed as an actual boolean I'd imagine - and then the UI here can choose what string to display if needed

Reply:
The reason for converting the value to uppercase first char is:

this.objectInfo.autoCreateIncrementalStatistics = true or false, 
we display this.booleanOptionsArray = [True, False], in createDropdown, the comparison is case sensitive, and the UI only shows the options when the array values must match with the passed value.

Reply:
Dropdown comparisons should be done using a non-localized value. You do this by using CategoryValues instead of just direct strings : https://github.com/microsoft/azuredatastudio/blob/main/src/sql/azdata.d.ts#L668

Reply:
If Erin confirm to change the DD to checkbox, This code will be gone! addressing remaining comments meanwhile.